actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsActor.java

```diff
@@ -498,7 +509,9 @@ private void orgConsumptionMetrics(Request actorMessage) {
       }
       String orgName = (String) orgData.get(JsonKey.ORG_NAME);
       String orgHashId = (String) orgData.get(JsonKey.HASHTAGID);
-      ProjectLogger.log("orgId" + orgHashId);
+      ProjectLogger.log(
```

Comment:
From:
"OrganisationMetricsActor:orgConsumptionMetrics orghashtag id "

To:
"OrganisationMetricsActor:orgConsumptionMetrics: org hash tag id = "